### PR TITLE
Logo, UI nits

### LINKED ISSUE #102: We need a logo for the favicon

### PR DESCRIPTION
**Pull request by goldsteinsveta**
- Closes #102
- Logo (works for 35px we have for it now :))
- Also layout, color and typography adjustments

Screenshot 2024-09-06 at 02 27 33
Screenshot 2024-09-06 at 02 27 24

### CODE REVIEW

File: src/components/MyDelegations.tsx [hunk 1]
```diff
@@ -86,46 +87,56 @@ export const MyDelegations = () => {
...
-                            <BadgeCent className="inline-block h-4 w-4 text-gray-500" />{' '}
-                            {planckToUnit(
-                              balance,
-                              assetInfo.precision,
-                            ).toLocaleString('en')}{' '}
-                            {assetInfo.symbol}
+                    <div className="mb-4 ml-12 pl-2">
+                      <Title variant={'h4'}>Delegating on tracks:</Title>
+                    </div>
+                    <ContentReveal noMaxHeight={true}>
```
Comment (Tbaut) [inline src/components/MyDelegations.tsx:110]: I've fixed this differently, by aggregating things in a clever way https://github.com/delegit-xyz/dashboard/pull/100

**Review by Tbaut — COMMENTED** [review]
Thank you so much for this. What's your fav logo? Mine is the 3, for the favicon in white on black.
image

Should we use this for the main logo at the top-left of the website too, instead of the Polkadot one?

There will be conflicts with my PR where I merge the related delegation info, I think it's important to show the current conviction and amounts rather than hiding it all.

File: src/components/MyDelegations.tsx (hunk 1 above)
Comment (goldsteinsveta) [reply src/components/MyDelegations.tsx:110]: 100 is yum. Merge that one and I can resolve conflicts with this PR later.

Comment (Tbaut) [reply src/components/MyDelegations.tsx:110]: Thanks, the conflicts are in :see_no_evil:

File: src/components/DelegationByAmountConviction.tsx [hunk 2]
```diff
@@ -17,7 +17,10 @@ const TrackDisplay = ({
   trackName: string
 }) => (
   <div className="capitalize" key={trackId}>
-    <Badge>{trackName}</Badge> /{trackId}
+    <Badge>{trackName}</Badge>
+    <span className="ml-2 border-l-2 pl-2 text-xs font-semibold text-slate-400">
+      {trackId}
```
Comment (Tbaut) [inline src/components/DelegationByAmountConviction.tsx:22]: super nice

File: src/pages/Delegate/index.tsx [hunk 3]
```diff
@@ -135,7 +135,12 @@ export const Delegate = () => {
           }
         />
       )}
-      <h1 className="flex-1 shrink-0 whitespace-nowrap font-unbounded text-xl font-semibold tracking-tight text-primary sm:grow-0">
+
+      <Link to="/home" className="flex items-center gap-2 text-primary">
+        <ArrowLeft className="h-4 w-4" />
```
Comment (Tbaut) [inline src/pages/Delegate/index.tsx:140]: that's great

**Pull request merged**